platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -128,47 +142,46 @@ public void init() throws Exception {
         return export(Paths.get(exportDirectory));
     }
 
-    BundleContext getBundleContext() {
-        return FrameworkUtil.getBundle(ConfigurationMigrationManager.class)
-                .getBundleContext();
-    }
+    private Collection<MigrationWarning> exportMigratable(Migratable migratable,
+            Path exportDirectory) throws IOException {
+        Stopwatch stopwatch = null;
 
-    private Collection<MigrationWarning> exportMigratables(Path exportDirectory) {
-        List<MigrationWarning> warnings = new LinkedList<>();
+        if (LOGGER.isDebugEnabled()) {
+            stopwatch = Stopwatch.createStarted();
+        }
+
+        if (DataMigratable.class.isInstance(migratable)) {
+            Path dataMigratableDirectory = exportDirectory.resolve(
```

Comment:
You shouldn't need to use `File.separator` here. `resolve` will hide all that OS specific crap from you.